src/components/Routes/PrivateRoute.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import {useSelector} from "react-redux";
+import {selectLoggedIn} from "../../redux/auth/selectors.js";
+import {Navigate} from "react-router-dom";
+import toast from "react-hot-toast";
+
+const PrivateRoute = ({ component, redirectTo = "/login" }) => {
```

Comment:
тут у пропси потрібно передати тільки children і в RouterSet приватні роути потрібно просто огорнути PrivateRoute
```suggestion
const PrivateRoute = ({ children}) => {
```